tools/cross/install_rootfs.sh

```diff
@@ -91,7 +91,7 @@ fi
 
 if [ -d "$__RootfsDir" ]; then
     if [ $__SkipUnmount == 0 ]; then
-        umount $__RootfsDir/*
+        umount -q $__RootfsDir/* || true
```

Comment:
just a question. :)
Why "|| true" is added?
It seems to ignore umount failure. right?

Reply:
Probably right :)